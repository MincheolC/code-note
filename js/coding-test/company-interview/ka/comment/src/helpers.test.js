const { commentTemplate } = require('./helpers')._;

describe('commentTemplate', () => {
  it('valid', () => {
    const comment = {
      id: 1472723537672,
      author: '김다음',
      comment: '첫 번째 댓글입니다.',
    };

    expect(commentTemplate(comment)).toMatchSnapshot();
  });
});
