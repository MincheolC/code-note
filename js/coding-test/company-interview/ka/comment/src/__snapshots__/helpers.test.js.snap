// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`commentTemplate valid 1`] = `
<div>
  <p
    class="author"
  >
    김다음
  </p>
  <p>
    Thu Sep 01 2016 18:52:17 GMT+0900 (대한민국 표준시)
  </p>
  <p>
    첫 번째 댓글입니다.
  </p>
</div>
`;
